app/views/unattended/partition_tables_templates/kickstart_custom.erb

```diff
@@ -19,35 +19,22 @@ clearpart --all --initlabel
   fstype = host_param('part_fstype') || 'xfs'
 -%>
 
-<% if @host.pxe_loader.include?('UEFI') -%>
-
+<% if @host.pxe_loader.include?('UEFI') or @host.pxe_loader.include?('BIOS') -%>
+  <%- if @host.pxe_loader.include?('UEFI') -%>
   part /boot/efi --fstype="efi" --ondisk=<%= dev %> --size=200 --fsoptions="umask=0077,shortname=efi"
-  <%- if (@host.operatingsystem.family == 'Redhat' && @host.operatingsystem.major.to_i >= 7 ) -%>
-    part /boot --fstype="<%= fstype %>" --ondisk=<%= dev %> --size=1024
-  <% else -%>
-    part /boot --fstype="ext4" --ondisk=<%= dev %> --size=1024
-  <% end -%>
-  part pv.01 --fstype="lvmpv" --ondisk=<%= dev %> --size=1024 --grow
-  volgroup vg_sda --pesize=4096 pv.01
-  logvol swap --fstype="swap" --size=<%= swap %> --name=lv_swap --vgname=vg_sda
-  logvol / --fstype="<%= fstype %>" --percent=<%= root %> --name=lv_root --vgname=vg_sda
-  <% if home != '0' -%>
-    logvol /home --fstype="<%= fstype %>" --percent=<%= home %> --name=lv_home --vgname=vg_sda
-  <% end -%>
-
-<% elsif @host.pxe_loader.include?('BIOS') -%>
+  <%- end -%>
 
   <%- if (@host.operatingsystem.family == 'Redhat' && @host.operatingsystem.major.to_i >= 7 ) -%>
-    part /boot --fstype="<%= fstype %>" --ondisk=<%= dev %> --size=1024
+  part /boot --fstype="<%= fstype %>" --ondisk=<%= dev %> --size=1024
   <% else -%>
-    part /boot --fstype="ext4" --ondisk=<%= dev %> --size=1024
+  part /boot --fstype="ext4" --ondisk=<%= dev %> --size=1024
   <% end -%>
   part pv.01 --fstype="lvmpv" --ondisk=<%= dev %> --size=1024 --grow
-  volgroup vg_sda --pesize=4096 pv.01
-  logvol swap --fstype="swap" --size=<%= swap %> --name=lv_swap --vgname=vg_sda
-  logvol / --fstype="<%= fstype %>" --percent=<%= root %> --name=lv_root --vgname=vg_sda
+  volgroup vg_<%= dev %> --pesize=4096 pv.01
```

Comment:
Would it make sense to define an additional variable for the volume group name?

Reply:
Yeah, I actually wanted to do that, but then you'd probably want to customize lv_swap lv_root and lv_home also.

I'm not up to speed on ruby templates, in jinja I'd do something like `{{ vgname | default('vg_' ~ dev) }}` If you can give me a hint on how to structure it I can create a issue/feature-request and implement it.

Reply:
Between line 14 and 20 those are defined. We use `|| 'default'`. In this case you would use `|| "vg_#{dev}"` to perform the string interpolation. Be sure it's below the definition of `dev`.

Reply:
Thanks :) Took a stab at customizing the names.